app/services/tims.rb

```diff
@@ -2,4 +2,26 @@
 # frozen_string_literal: true
 
 class Tims
+
+  def get_disruptions
+    download_and_filter_disruption.map{ |disruption| Disruption.new(*first_divided(disruption["CauseArea"]["DisplayPoint"]["Point"].values)) }
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.